src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball = new Ball();
+        for (int i = 0; i < ball.randomBalls; i++) {
+            ball = lottery.getRandomBall();
+            System.out.println(ball.toString());
```

Comment:
`System.out.println` calls method `toString()` under the hood by default.
and you can optimize your code 
```suggestion
            System.out.println(lottery.getRandomBall());
```